fitbenchmarking/utils/options.py

```diff
@@ -139,7 +141,8 @@ class Options:
          'matlab_opt': ['levenberg-marquardt', 'trust-region-reflective'],
          'matlab_stats': ['Levenberg-Marquardt'],
          'minuit': ['minuit'],
-         'ralfit': ['gn', 'gn_reg', 'hybrid', 'hybrid_reg'],
+         'ralfit': ['gn', 'gn_reg', 'hybrid', 'hybrid_reg',
+                    'newton', 'newton_reg'],
```

Comment:
How do you decide which minimizers should be the default?

Reply:
`newton` and `newton_reg` take an age to run (by design).  I think this means you should actively think you want to test with them, rather than think "I'll run RALFit" and then grumble that it takes forever.  I'll update the docs for RALFit to explain this.